Packs/AWS-SecurityHub/ReleaseNotes/1_1_16.md

```diff
@@ -0,0 +1,5 @@
+
+#### Integrations
+- **AWS - Security Hub**
+- Updated the Docker image to: *demisto/boto3py3:1.0.0.29538*.
+- Fixed an issue where **list_members_command** failed due to response containing values of type datetime.
```

Comment:
Get yourself familiarized with the documentation styling guidelines https://xsoar.pan.dev/docs/documentation/release-notes#entity-styling
```suggestion
- Fixed an issue where the ***aws-securityhub-list-members*** failed to parse API responses that contained datetime objects.
- Updated the Docker image to: *demisto/boto3py3:1.0.0.29538*.

```